Assets/LoggingManager/LogStore.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Boo.Lang.Runtime;
+using UnityEngine;
+
+
+public enum LogType
+{
+    Normal,
+    Meta
+}
```

Comment:
In the LoggingManager, the user can't decide to use Meta logs or regular logs. He can only create a meta log by calling the `CreateLog()` [method](https://github.com/Vincent-SD/ArduinoLogger/blob/06cbd9a1d1288205624b9d286d0c12b86799b5c6/Assets/LoggingManager/LoggingManager.cs#L114). 

I think I misunderstood something with the use of the LogType. 
Can the user have access to this ? If so, what is the purpose ? 

Maybe is it possible to keep this enum like this and change the way we write to the file. Instead of having only one method to write all the lines in the file, we could also add a method that adds a custom line to a file.

Reply:
@Vincent-SD users of LoggingManager don't access LogStore directly. But LogStore has a public API and needs to be maintained by other developers in the future. Imagine a future developer glancing at `LogType` and asking.. "Normal"...what does normal mean actually? What is "Normal"?

My main feedback to you here is, that it is better to use the enum labels, to be descriptive of the _behavior_ of what the options do. What does "Normal LogType" do in practice? What is a normal logtype? What does a "Meta logtype" do in practice? How does it behave?

In addition to a more descriptive label, I think we should add an in-line code comment here describing the behavior in detail. That's all. 🍸